op-dispute-mon/mon/withdrawals.go

```diff
@@ -48,6 +62,16 @@ func (w *WithdrawalMonitor) validateGameWithdrawals(game *types.EnrichedGameData
 			divergent++
 			w.logger.Error("Withdrawal request amount does not match credit", "game", game.Proxy, "recipient", recipient, "credit", game.Credits[recipient], "withdrawal", game.WithdrawalRequests[recipient].Amount)
 		}
+
+		if withdrawalAmount.Amount.Cmp(big.NewInt(0)) > 0 && w.honestActors.Contains(recipient) {
+			if time.Unix(withdrawalAmount.Timestamp.Int64(), 0).Add(game.WETHDelay).Before(now) {
+				// Credits are withdrawable
+				total := honestWithdrawableAmounts[recipient]
+				total = new(big.Int).Add(total, withdrawalAmount.Amount)
+				honestWithdrawableAmounts[recipient] = total
+				w.logger.Warn("Found unclaimed credit", "recipient", recipient, "game", game.Proxy, "amount", withdrawalAmount.Amount)
```

Comment:
nit: This logs as soon as it's withdrawable and the op-challenger does take a little bit of time to send withdraw transaction. Which is OK as the dispute-mon loop doesn't run that often to catch this at the exact time the bond is withdrawable.
Consider adding a 24 second grace period before logging here.

Reply:
Yeah we've consistently done that with op-dispute-mon - it will log a warning immediately and the delay is configured in the alerts. It seems to work pretty well since you can clearly see the update cycles in the logs as well and see which alerts are coming from each update and when the update cycle then is giving no warnings etc.